harness/features/allVariables.local.test.ts

```diff
@@ -53,12 +47,30 @@ describe('allVariables Tests - Local', () => {
             await waitForRequest(
                 scope,
                 interceptor,
-                1000,
+                3000,
                 'Config request never received!',
             )
         })
 
         it('should return a variable map for a bucketed user', async () => {
+            const client = new LocalTestClient(sdkName)
+            scope
+                .get(
+                    `/client/${client.clientId}/config/v1/server/${client.sdkKey}.json`,
+                )
+                .reply(200, config)
+
+            if (hasCapability(sdkName, Capabilities.sdkConfigEvent)) {
```

Comment:
will wrapping this `scope.post` inside the IF statement result in any other test issues (i.e. when other tests are mocking out the response to the `/events/batch` endpoint)

Reply:
ohh so this one specifically we were setting up two `LocalTestClient` objects for the `should return an empty object if client is not initialized` test above before, which was causing issues when testing these changes as the client setup in the `beforeEach` would never get used. So it made more sense to setup the two `LocalTestClient` in the two tests here.

Reply:
Got it, sounds good!